reference/5.1/Microsoft.PowerShell.Core/About/about_Arrays.md

```diff
@@ -891,11 +891,17 @@ faster, especially for large arrays.
 
 ## Arrays of zero or one
 
-Beginning in Windows PowerShell 3.0, a collection of zero or one object has
-the Count and Length property. Also, you can index into an array of one
+Beginning in Windows PowerShell 3.0, a collection of zero or one object has the
+**Count** and **Length** properties. Also, you can index into an array of one
 object. This feature helps you to avoid scripting errors that occur when a
 command that expects a collection gets fewer than two items.
 
+> [!TIP]
+> In Windows PowerShell, objects created by casting a **Hashtable** to
+> `[pscustomobject]` are an exception and return `$null` for the **Count** and
+> **Length** properties, though they do support the array indexing
+> functionality.
+
```

Comment:
I am unsure of the wording here but I do feel it is useful to call out the behavior here as well as in `about_PSCustomObject` as it directly contradicts the stated documentation.